src/_base/docker/image/console/root/lib/task/build/backend.sh.twig

```diff
@@ -2,8 +2,11 @@
 
 function task_build_backend()
 {(
+    set -e -o pipefail
+
     cd {{ @('backend.path') }}
-    
+
+    # shellcheck disable=SC2160
     if [ ! {{ @('backend.build.when')|raw }} ]; then
```

Comment:
this seems to suggest that we'll have issues with when: "false" as I presume this was flagged because [ ! true ] rendered?

Reply:
in heinsight, incorporating the [ ] into the when prob would have been better

Reply:
Yeah, it's as you suspect - `false` values get a complaint that you can just do `if ! false; then` instead of `if [ ! false ]; then`